src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -907,44 +911,49 @@ int doLastBlock(GCMOperation op, ByteBuffer buffer, ByteBuffer src,
          * and if dst will overwrite src data before src can be processed.
          * If so, make a copy to put the dst data in.
          */
+        @SuppressWarnings("try")
```

Comment:
After looking at the implementation some more, I'm not sure this need fixing? E.g. this method is just using the address to compute some overlap - and return a buffer sliced accordingly. There's no access to the buffer data (except for the last part which does a `put`). The access will fail if the session is closed from underneath. I don't think this can crash the VM (in fact this code did not have a reachability fence to begin with).

Reply:
This `@SuppressWarnings` annotation is no longer needed:
```suggestion
```